source/macos/AdaptiveCards/AdaptiveCards/Views/ACRButton.swift

```diff
@@ -4,33 +4,18 @@ enum ActionStyle: String {
     case positive
     case `default`
     case destructive
-    case none
+    case darkMode
```

Comment:
This is not required. `darkMode` is not a buttonStyle. It's just a variation of default button.